addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -54,7 +54,7 @@ test("many2one in list view", async () => {
     // click on first employee avatar
     await contains(".o_data_cell .o_m2o_avatar > img:eq(0)").click();
     await waitFor(".o_avatar_card");
-    expect(".o-mail-avatar-card-name").toHaveText("Mario");
+    await waitFor(".o-mail-avatar-card-name:contains('Mario')");
```

Comment:
`toHaveText` did exact matching, which is better.
Equivalent `contains`  should have exact match regex (hopefully soon we should have the exact match selector too).
`:contains(/^Mario$/)`